Makefile

```diff
@@ -0,0 +1,10 @@
+test:
+	PYTHONPATH=".:./tests" pytest --capture=sys --cov-report term-missing --cov=sensu_handler_pagerduty_alternative ${PYTEST_FLAGS}
+
+style: flake8 pylint
+
+pylint:
+	pylint --ignore-patterns="^.*pb2.*\.py, setup.py" sensu_handler_pagerduty_alternative.py
+
+flake8:
+	flake8 --exclude="*pb*py, tests, setup.py" --max-line-length=100
```

Comment:
All the targets in this `Makefile` run commands that don't build output files, so it's worth adding `.PHONY: test`, etc. for each of them to ensure the targets always run even if a file happens to exist with the same name at some point. It's been oddly frustrating to run into sometimes, so worth explicitly marking them as such.

Reply:
Done!